docs/changelog/95175.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 95175
+summary: Add model downloader x-pack core changes
```

Comment:
We really want one release note for the model downloader functionality. It's probably best to mark this PR `>non-issue` and add the release note in one of the followup PRs.